### PR TITLE
Doc-6645

### PR DESCRIPTION
**Pull request by tamaskiss00**
Some link changes in the manpages: xref to <a>
added the xsd link to the html tag

### CODE REVIEW

File: Content/Guides/manpages/pdbtool.1.html [hunk 1]
```diff
@@ -295,9 +295,8 @@ <h6>Files</h6>
 		</div>
 		<div>
 			<h6>See also</h6>
-			<p><MadCap:xref href="https://www.syslog-ng.com/"><span style="color: #04aada;" class="mcFormatColor">The syslog-ng
-					Administrator Guide</span>
-				</MadCap:xref>
+			<p><a href="https://www.syslog-ng.com/"><span style="color: #04aada;" class="mcFormatColor">The syslog-ng
```
Comment (c4tc4t) [inline Content/Guides/manpages/pdbtool.1.html:298]: Is the gap here lots of whitespaces, or does it only look like that in this merge request view?  (The same question for all occurrences.)

File: Content/Guides/shared/chunk/pe-para-chain-hostnames.htm [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
     <head>
         <meta name="description" content="" />
     </head>
-    <body>The <span class="Code">chain-hostnames()</span> option of syslog-ng can interfere with the way <MadCap:variable name="General.abbrev" xmlns:MadCap="http://www.madcapsoftware.com/Schemas/MadCap.xsd"></MadCap:variable> counts the log source hosts, causing syslog-ng to think there are more hosts logging to the central server, especially if the clients sends a hostname in the message that is different from its real hostname (as resolved from DNS). Disable the <b class="Code">chain-hostnames()</b> option on your log sources to avoid any problems related to license counting.</body>
+    <body>The <span class="Code">chain-hostnames()</span> option of syslog-ng can interfere with the way <MadCap:variable name="General.abbrev"></MadCap:variable> counts the log source hosts, causing syslog-ng to think there are more hosts logging to the central server, especially if the clients sends a hostname in the message that is different from its real hostname (as resolved from DNS). Disable the <b class="Code">chain-hostnames()</b> option on your log sources to avoid any problems related to license counting.</body>
```
Comment (c4tc4t) [inline Content/Guides/shared/chunk/pe-para-chain-hostnames.htm:6]: `The <span class="Code">chain-hostnames()</span> option can interfere with the way <MadCap:variable name="General.abbrev"></MadCap:variable> counts the log source hosts. As a result, <MadCap:variable name="General.abbrev"></MadCap:variable> falsely perceives several hosts logging to the central server`

File: Content/Guides/manpages/pdbtool.1.html (hunk 1 above)
Comment (tamaskiss00) [reply Content/Guides/manpages/pdbtool.1.html:298]: fixed in every manpage

**Review by c4tc4t — APPROVED** [review]
Approved.

**Pull request merged**